Ninjathulhu/Ninjathulhu.Game/Entity.cs

```diff
@@ -68,6 +68,30 @@ public class EntityProperties
         public Dictionary<Type, ComponentProperties> FromPrefab = new Dictionary<Type, ComponentProperties>();
 
         public Dictionary<Type, ComponentProperties> FromSpawn = new Dictionary<Type, ComponentProperties>();
+
+        private static object Get(Dictionary<Type, ComponentProperties> properties, Type componentType, string propertyName)
```

Comment:
Yeah, I figured I'd probably stick an `/Entity` subdir in there, maybe see if I can organize `Ninjathulhu.Game` into "common components" and "game components" or whatever. This is all small enough that it should be easy enough to change.